### PR TITLE
fix(learn): fixed typo "in the left" should be "at the left"

### PR DESCRIPTION
**Pull request by farjadfazli**
Checklist:

- [ x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [ x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.english.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ forumTopicId: 16784
 ## Description
 <section id='description'>
 There are many <dfn>comparison operators</dfn> in JavaScript. All of these operators return a boolean <code>true</code> or <code>false</code> value.
-The most basic operator is the equality operator <code>==</code>. The equality operator compares two values and returns <code>true</code> if they're equivalent or <code>false</code> if they are not. Note that equality is different from assignment (<code>=</code>), which assigns the value at the right of the operator to a variable in the left.
+The most basic operator is the equality operator <code>==</code>. The equality operator compares two values and returns <code>true</code> if they're equivalent or <code>false</code> if they are not. Note that equality is different from assignment (<code>=</code>), which assigns the value at the right of the operator to a variable at the left.
```
Comment (Sky020) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.english.md:12]:
```suggestion
The most basic operator is the equality operator <code>==</code>. The equality operator compares two values and returns <code>true</code> if they're equivalent or <code>false</code> if they are not. Note that equality is different from assignment (<code>=</code>), which assigns the value on the right of the operator to a variable on the left.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
This is what makes sense to me.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.english.md (hunk 1 above)
Comment (farjadfazli) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.english.md:12]: Yes that works too!

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by moT01 — APPROVED** [review]
LGTM @farjadfazli 🎉 Congratulations on your first contribution to freeCodeCamp 🎉 

For future reference, it's good to give your branch a more descriptive name and start it with a scope. See `#3` just below [here in the docs](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-setup-freecodecamp-locally.md#making-changes-locally)

**Comment by farjadfazli** [comment]
Thank you! I will keep the branch naming in mind for next time 👍

**Pull request merged**